app/actions/remote/session.ts

```diff
@@ -49,6 +50,12 @@ export const completeLogin = async (serverUrl: string, user: UserProfile) => {
 
     await DatabaseManager.setActiveServerDatabase(serverUrl);
 
+    // Set push proxy verification
+    const ppVerification = EphemeralStore.pushProxyVerification[serverUrl];
+    if (ppVerification) {
+        operator.handleSystem({systems: [{id: SYSTEM_IDENTIFIERS.PUSH_VERIFICATION_STATUS, value: ppVerification}], prepareRecordsOnly: false});
```

Comment:
I wonder if we should add this in line 63 instead of calling `handleSystem` twice